<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit_position_X"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_X_move_n">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_step_X"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_X_move_p">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_target_X"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_position_Y"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Y_move_n">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_step_Y"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Y_move_p">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_target_Y"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit_position_Z"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Z_move_n">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_step_Z"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Z_move_p">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_target_Z"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Joystick">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Joystic OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_HomeX">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_HomeY">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_HomeZ">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleZ">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius [mm]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleRadius">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay [s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleDelay">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of segments</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CircleSegments">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_Circle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ReadCenter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Read center</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TraceCircle">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Trace circle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracing progress</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_step_X</tabstop>
  <tabstop>lineEdit_step_Y</tabstop>
  <tabstop>lineEdit_step_Z</tabstop>
  <tabstop>pushButton_X_move_p</tabstop>
  <tabstop>lineEdit_position_X</tabstop>
  <tabstop>lineEdit_target_X</tabstop>
  <tabstop>lineEdit_position_Y</tabstop>
  <tabstop>pushButton_Y_move_n</tabstop>
  <tabstop>pushButton_Close</tabstop>
  <tabstop>pushButton_Y_move_p</tabstop>
  <tabstop>lineEdit_target_Y</tabstop>
  <tabstop>lineEdit_position_Z</tabstop>
  <tabstop>pushButton_Z_move_n</tabstop>
  <tabstop>pushButton_X_move_n</tabstop>
  <tabstop>pushButton_Z_move_p</tabstop>
  <tabstop>lineEdit_target_Z</tabstop>
  <tabstop>pushButton_Connect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_X_move_p</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_X</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Y_move_p</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_Y</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_X_move_n</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_X</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Y_move_n</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_Y</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Z_move_p</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_Z</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Z_move_n</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_target_Z</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
